src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -138,6 +138,9 @@ h1, h2, h3, h4 {
 .docblock h3, .docblock h4, h5, h6 {
 	margin: 15px 0 5px 0;
 }
+.docblock table td, .docblock table th {
+	border-color: var(--border-color);
```

Comment:
This should be combined with the rules that set the border for these elements in the first place.

```css
.docblock table td {
	padding: .5em;
	border: 1px dashed;
}

.docblock table th {
	padding: .5em;
	text-align: left;
	border: 1px solid;
}
```

Reply:
Yes absolutely. Don't know how I didn't think about it...